misc/python/materialize/setup.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Josh Wills. All rights reserved.
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the LICENSE file at the
+# root of this repository, or online at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from distutils.core import setup
+
+setup(
+    name="mz_test_base",
```

Comment:
It seems as if this subdirectory includes a bunch of tools beyond testing -- does it make sense to name the installation something different? I would suggest perhaps `materialize`, since that's the name of the top-level package we're installing anyway.